static/usr/share/subiquity/console-conf-write-login-details

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+
+pretty_name="$( awk -F "=" '$1 = /PRETTY_NAME/ {print $2;}' /usr/lib/os-release | sed 's/"//g')"
+tty_name=$(tty | cut -c 6-)
+
+get_first_ip() {
+  for iface in $(ip -br address | awk '{print $1}')
+  do
+    if test -n ${iface} && ip addr show dev ${iface} >/dev/null 2>&1; then
+      if test -e /sys/class/net/${iface}/device; then
+        local ip_address="$(ip -4 -br addr show dev ${iface} | awk '{print$3}' | sed 's|\(.*\)/.*|\1|')"
+        if [ -n "${ip_address}" ]; then
+          echo "${ip_address}"
+          return 0
+        fi
+      fi
+    fi
+  done
+  echo "<no ip address>"
+  return 2
+}
+
+print_ssh_commands() {
+  # take any user with non zero
+  for h in /root $(ls /home/ | awk '{print "/home/"$1}')
```

Comment:
`/home/*`?

Reply:
yes, that's for the screen where user's login information is printed after boot

Reply:
ah, nvm :)

Reply:
I feel like reading through `getent passwd`, then selecting any user with a shell that is not nologin, and get their home dir from there would be a bit better actually.